README.md

```diff
@@ -49,7 +49,7 @@ data class Project(val name: String, val language: String)
 fun main() {
     // Serializing objects
     val data = Project("kotlinx.serialization", "Kotlin")
-    val string = Json.encodeToString(data)  
+    val string = Json.encodeToString(Project.serializer(), data)  
```

Comment:
```suggestion
    val string = Json.encodeToString<Project>(data)  
```
Would be more consistent I think.

Reply:
Yeah sorry, I had problems using it and needed to provide the serialiser. But that's probably only because I customised it with @Serializable(with = MyProjectSerializer::class). Feel free to ignore/reject.